packages/subgraph-service/contracts/SubgraphService.sol

```diff
@@ -22,9 +22,14 @@ import { PPMMath } from "@graphprotocol/horizon/contracts/libraries/PPMMath.sol"
 import { Allocation } from "./libraries/Allocation.sol";
 import { LegacyAllocation } from "./libraries/LegacyAllocation.sol";
 
+import { ProvisionManager } from "@graphprotocol/horizon/contracts/data-service/utilities/ProvisionManager.sol";
+
+
+
 contract SubgraphService is
     Initializable,
     OwnableUpgradeable,
+    ProvisionManager,
```

Comment:
We shouldn't need to inherit `ProvisionManager` here as we are already inheriting `DataService` which in turn inherits `ProvisionManager` 😄